framework/src/Volo.Abp.BlazoriseUI/Components/PageHeader.razor

```diff
@@ -0,0 +1,35 @@
+﻿<Row Class="entry-row">
+    <Column ColumnSize="ColumnSize.IsAuto">
+        <h1 class="content-header-title">@Title</h1>
+    </Column>
+    @if (BreadcrumbItems.Any())
+    {
+        <Column ColumnSize="ColumnSize.IsAuto.OnWidescreen" Class="pl-lg-0">
```

Comment:
Instead of using class "pl-lg-0", I would write `Padding="Padding.Is0.FromLeft.OnLarge"`. In fact, the best practice would be to use as many Blazorise utilities as possible

Reply:
Done.